### PR TITLE
raidboss: Improvements to FRU P2 timeline

### PR DESCRIPTION
**Pull request by JLGarber**
This has been tested in-game. I adjusted some timings to better match what I saw across my logs to match `test_timeline` and added the remaining `targetable` entries. I also combined the Shining and Frost armor entries since those always go off simultaneously but the order in the log is random. Finally, I filled out some more data on the P2 abilities in the ability list.

Timings for intermission and beyond have not been modified.

### CODE REVIEW

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:74]: Would it be worth shortening this to `Axe/Scythe Kick` in `replaceText`?

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:75]: I'm pretty sure I have a log of this ability also coming from combatants that are just `Fatebreaker`.  Given the issues with combatant re-use., I'm wondering if maybe removing the `source` property is the way to go?

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (kshman) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:76]: I found another source
AOEActionEffect 16:4002699C:Usurper of Frost:9D07:Frigid Stone:
AOEActionEffect 16:40013FF1:Fatebreaker's Image:9D07:Frigid Stone:

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:77]: Ibid.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:76]: I've seen multiple source names on this one too.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:79]: Ibid.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:80]: Ibid.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:83]: Ibid.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:107]: I wonder if it might be worthwhile to have the 344.0 be `Bright Hunger (solo towers)` and the 358.8 be `Bright Hunger (center tower)`, just for clarity?  Feel free to disregard though, they're probably far enough apart to not be confusing.

Also wondering about the whether it would be worth including the `Burst` (9D1B) from the `Holy Light` combatants (the orbs on the ground).  The timing on those is actually probably a Helpful Thing.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:138]: If you want to do a pre-sync here before Junction so it will actually show up correctly on the timeline in advance, you could sync to the `WasDefeated` line for the Ice Veil, e.g.:
`25|[ts]|40008D78|Ice Veil|40008D53|`

It happens 11.2s before the Junction ability lines, so `488.8`.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (JLGarber) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:74]: Can't hurt.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (JLGarber) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:76]: Per wexxlee's suggestions below, I think we're maybe just better off not having a source here.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (JLGarber) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:107]: I suppose it's reasonable enough, yes. Agreed on adding the `Burst` entries, the timing there is useful.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (JLGarber) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:138]: That's fine, I'll add that. I'd prefer not to touch more on the intermission than that, since I was focused specifically on the main P2 section here, and my group still hasn't got past Ultimate Relativity.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt
Comment (wexxlee) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt:142]: I think this is `target: Ice Veil`

**Review by wexxlee — APPROVED** [review]
Approving now, just see my added comment about the Phase 3 sync.  Also, it might've gotten lost in the comments because GH is annoying about tagging the right line for review comments, but these lines also have varying source names and we might want to remove the source prop.  Leave it up to you though:

```
247.2 "Frigid Stone" Ability { id: "9D07", source: "Usurper of Frost" }
247.6 "Icicle Impact" Ability { id: "9D06", source: "Usurper of Frost" }
251.5 "Heavenly Strike" Ability { id: "9D0F", source: "Usurper of Frost" }
251.6 "Icicle Impact" Ability { id: "9D06", source: "Usurper of Frost" }
255.5 "Icicle Impact" Ability { id: "9D06", source: "Usurper of Frost" }
```

**Comment by JLGarber** [comment]
Ah, thanks for taking care of that. I was out with family all day and wasn't able to check in on things.

**Pull request merged**
